pkg/inject/sidecar.go

```diff
@@ -367,6 +391,20 @@ func HasJaegerAgent(dep *appsv1.Deployment) (bool, int) {
 	return false, -1
 }
 
+// IsJaegerPortAvailable checks whether deployment is already using some port
+func IsJaegerPortAvailable(port int32, dep *appsv1.Deployment) bool {
+	// this pod is annotated, it should have a sidecar
+	// but does it already have one?
```

Comment:
Is this comment part of a copy/paste? Seems like it belongs to some other place :-)

Reply:
:man_facepalming: